src/components/recs/recorded-games.tsx

```diff
@@ -46,7 +59,7 @@ export default function RecordedGames() {
           setQuery(search);
           setSelectedBuild(parseInt(buildNumber));
         } else {
-          mappedFilters = { buildNumbers: [builds[0]] }; // filter by latest build on load
+          mappedFilters = { buildNumbers: [builds[1]] }; // filter by latest build on load
```

Comment:
Does this actually set it to the latest build or is this a hard coded way to get around not showing the red bull patch first? I ask because there is literally a new patch today and I would expect it to be build[0]